File: lecandy.py
t=int(input())
while t>0:
    x=0
    n,c=input().split(" ")
    n=int(n)
    c=int(c)
    A=list(map(int,input().split(" ")))
    i=0
    while i<n:
        x+=A[i]
        i+=1
    if(x<=c):
        print("Yes")
    else:
        print("No")
    t-=1

File: LECANDY.c
#include <stdio.h>
int main()
{
int t,i,n,j,c,x,a[100];
scanf("%d",&t);
for(i=0;i<t;i++)
{
x=0;
scanf("%d %d",&n,&c);
for(j=0;j<n;j++)
{
scanf("%d",&a[j]);
x+=a[j];
}
if(x<=c)
printf("Yes\n");
else
printf("No\n");
}
return 0;
}
